common/base_snapshot_check_revert.yml

```diff
@@ -2,20 +2,23 @@
 # SPDX-License-Identifier: BSD-2-Clause
 ---
 # Check if base snapshot exists when parameter is undefined
-- block:
-    - include_tasks: ../../common/vm_check_snapshot_exist.yml
+- name: "Check if base snapshot exists"
+  block:
+    - name: "Check if snapshot {{ base_snapshot_name }} exists"
+      include_tasks: vm_check_snapshot_exist.yml
       vars:
         snapshot_name: "{{ base_snapshot_name }}"
       when: >
         (new_vm is undefined) or
         (not new_vm | bool)
 
-    - name: Set fact of base snapshot existence
+    - name: "Set fact of base snapshot existence"
       ansible.builtin.set_fact:
         base_snapshot_exists: "{{ true if snapshot_exist is defined and snapshot_exist | bool else false }}"
```

Comment:
The value can be simplified as "{{ snapshot_exist | default(false) }}"

Reply:
Updated.